packages/web/src/components/track/MetadataLabel.tsx

```diff
@@ -0,0 +1,22 @@
+import { PropsWithChildren } from 'react'
+
+import { Text } from '@audius/harmony'
+
+type MetadataLabelProps = PropsWithChildren<{
+  label: string
+}>
+
+export const MetadataLabel = (props: MetadataLabelProps) => {
+  const { label, children } = props
+
+  return (
+    <Text>
```

Comment:
oh cool so this is text, maybe consolidate implementations and naming a bit with mobile?

Reply:
oh i think i understand why mobile has Flex, cause of the image right?

Reply:
yeah it's annoying but so it goes